### PR TITLE
make sublime-text3 restricted

### PR DESCRIPTION
**Pull request by classabbyamp**
#### Testing the changes
- I tested the changes in this PR: **briefly**

from the [EULA](https://www.sublimehq.com/eula) 2b (emphasis mine):
> **You may not distribute** or sell license keys or **the SOFTWARE PRODUCT to third parties**. Licenses will be revoked if distributed or sold to third parties.

I followed the example of sublime-merge -> sublime-merge-bin

### CODE REVIEW

File: srcpkgs/sublime-text3-bin/template [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# Template file for 'sublime-text3-bin'
+pkgname=sublime-text3-bin
+version=3211
+revision=1
```
Comment (classabbyamp) [inline srcpkgs/sublime-text3-bin/template:4]: not quite sure how the version/revision should be done here and on the metapackage. open for advice.

File: srcpkgs/sublime-text3-bin/template [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+depends="libpng gtk+ hicolor-icon-theme desktop-file-utils"
+short_desc="Sophisticated text editor for code, markup and prose"
+maintainer="Andrea Brancaleoni <miwaxe@pompel.me>"
+license="custom:EULA"
+homepage="http://www.sublimetext.com/3dev"
+_license_checksum=33929b71625d13dacf2a0a5853171b9c04058f71e2955ee660b8d0f8dda45ed1
+repository="nonfree"
+restricted=yes
+nopie=yes
+replaces="sublime-merge>0"
```
Comment (classabbyamp) [inline srcpkgs/sublime-text3-bin/template:17]: this didn't work in my testing, but it's how `sublime-merge{,-bin}` did it. what's the proper way?

**Comment by paper42** [comment]
I would just mark sublime-text3 as restricted. This way it will be the same as `sublime-text4`.

**Comment by classabbyamp** [comment]
adjusted

**Comment by classabbyamp** [comment]
should it be revbumped?

**Comment by paper42** [comment]
> should it be revbumped?

yes

File: srcpkgs/sublime-text3/template [hunk 3]
```diff
@@ -16,12 +23,23 @@ else
...
+
+	filesum="$(xbps-digest EULA)"
+	if [ "$filesum" != "$_license_checksum" ]; then
+		msg_error "SHA256 mismatch for EULA:\n$filesum\n"
+	fi
+}
 
 do_install() {
+	vlicense EULA
+	rm eula EULA
```
Comment (paper42) [inline srcpkgs/sublime-text3/template:42]: Why is this necessary?

Comment (classabbyamp) [reply srcpkgs/sublime-text3/template:42]: a couple lines down it runs `cp --preserve=mode -r ./* "${DESTDIR}/usr/lib/sublime_text", which catches `eula` (the unprocessed HTML) and `EULA` (the text-only EULA) and puts them in the wrong place

File: srcpkgs/sublime-text3-bin/template (hunk 2 above)
Comment (Chocimier) [reply srcpkgs/sublime-text3-bin/template]: set `sublime-text3>0` and not `sublime-merge>0`

Comment (classabbyamp) [reply srcpkgs/sublime-text3-bin/template]: unneeded, I changed approach: https://github.com/void-linux/void-packages/pull/35896#issuecomment-1059963424

**Comment by Chocimier** [comment]
Sublime-merge was turned into empty package in order to remove from all the mirrors and from repodata, what anticipates "why can't fetch that" complaints.

**Review by paper42 — APPROVED** [review]

**Comment by Chocimier** [comment]
`repository=nonfree` need to stay to remove previous pakcage from repo, `archs` can be kept too. Otherwise fine.

**Comment by classabbyamp** [comment]
got it, should be all good now
